apps/rule-manager/test/db/RuleManagerSpec.scala

```diff
@@ -88,6 +88,7 @@ class RuleManagerSpec extends FixtureAnyFlatSpec with Matchers with AutoRollback
       maybeCheckerRule shouldBe Left(
         List(
           FormError("invalid-pattern", List("error.required"), List()),
+          FormError("invalid-pattern", List("Error parsing the XML: Premature end of file."), List()),
```

Comment:
Just to clarify - where/how would this error be generated? What is it about the example `rule` in this test that would generate this specific error?

Reply:
I think this is happening because there's no `pattern` defined in the rule – so as well as giving an error that the field is required, the validation also complains that there's nothing to parse.